themes/light/templates/vue-components/file-browser.mako

```diff
@@ -262,11 +267,22 @@ Vue.component('file-browser', {
             return resultField;
         };
 
-        $(this.$el).children('input').fileBrowser({
+        $(this.$refs.locationInput).fileBrowser({
```

Comment:
This was linking the file browser to 3 elements including the browse button, causing some issues with the autocomplete, if I remember the issue correctly.